pom.xml

```diff
@@ -451,13 +451,6 @@
                         <tag>${revision}${changelist}</tag>
                     </configuration>
                 </plugin>
-                <plugin>
```

Comment:
Is this intentional? Just making sure, it seems unrelated to the PR.

Reply:
it's unrelated, looks like a duplicate entry with the one above